website/webpack.plugin.js

```diff
@@ -0,0 +1,118 @@
+// const MonacoWebpackPlugin = require('monaco-editor-webpack-plugin');
```

Comment:
I still have to go through and remove all these overrides & customizations. Will do once I get Node set up in the Windows area of my new laptop...